README.md

```diff
@@ -1,24 +1,186 @@
+```
+
+### Install
+
+Install the project dependencies with:
+
+```sh
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm run build
+```
+
+### Run tests
+
+To run tests, run the following command:
+```sh
+  npm start
+```
+- Open your web browser and visit http://localhost:8080.
+- You should now see socials application running in your browser.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Kevin Cruz**
+
+- GitHub: [EvansSnave](https://github.com/EvansSnave)
+- LinkedIn: [kevin-cruz](https://linkedin.com/in/kevin-cruz-25159a201)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏🏻 Acknowledgements <a name="acknowledgements"></a>
+
+I'd like to express my gratitude to everyone who made this project possible.
+
+<!--- Future Features ---->
+
+## <a href="#future-features"></a>🔮 Future Features
+- Basic UI design
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions are welcome! If you find any bugs or want to enhance the project with new features, please follow these steps:
+
+1. Fork the repository.
+2. Create a new branch for your feature or bug fix.
+3. Make the necessary changes and commit them.
+4. Push your changes to your forked repository.
+5. Submit a pull request describing your changes.
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please consider supporting us! Your support helps us to continue developing and improving our project for everyone to enjoy. Whether it's a small donation, sharing the project with your friends, or contributing your skills and expertise, every little bit makes a huge difference. With your support, we can create something truly amazing together. Thank you for being a part of our community and helping us to make a difference!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ -->
+
+## ❓ FAQ <a name="faq"></a>
+- **How do I set up the project?**
+
+  - Clone the repository from GitHub, and see the [Setup](#setup) section above for details.
+
+- **How can I contribute to the project?**
+
+  1. Fork the repository on GitHub.
+  2. Create a new branch for your changes using git checkout -b my-feature-branch.
+  3. Make your changes and commit them with descriptive commit messages.
+  4. Push your changes to your forked repository.
+  5. Submit a pull request to the original repository.
+  6. Wait for feedback and iterate on your changes as needed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] Please 🙏🏻  add the `MIT` license properly because including the **MIT** *License* in your `GitHub` repository **README** is crucial for several reasons. Firstly, the **MIT** License is a permissive `open-source` license that grants users the freedom to use, modify, and distribute the software without legal restrictions. By choosing the *MIT* License 🗒️, you are encouraging collaboration, inviting other developers to contribute, and fostering a vibrant open-source community. Additionally, using the *MIT* License helps protect your work by clearly defining the terms and conditions under which it can be used, providing a level of legal 🚨 clarity and ensuring that proper attribution is given. Furthermore, the widespread popularity and familiarity of the *MIT* License make it easy for others to understand and comply with the licensing requirements, which ultimately promotes the adoption and usage of your software 💻 .